### ISSUE TITLE
square icon

### ISSUE THREAD

**Issue by Botspot** [alienzhangyw/BlockPi#11]
Please create an icon for BlockPi that is **square**.
Your current logo is a rectangle, and it does not look very good when scaled to a square.
![v2-03e04912b8ce7e16236d7d08c7747ccb_xs](https://user-images.githubusercontent.com/54716352/116485789-b3f61b80-a851-11eb-9925-8d3c02e4c78e.png)

**Comment by alienzhangyw** [comment]
I have provided squre icons in https://github.com/alienzhangyw/BlockPi/tree/master/build/icons.
The icon in app page is a svg with colored text fonts.

**Comment by Botspot** [comment]
Okay, let me rephrase: please create an icon that is **_recognizable_** when as small square. It should be easy to recognize at 24x24 pixels

Your app is being added to the [Pi-Apps Raspberry Pi app store](https://github.com/Botspot/pi-apps). Currently, your icon is not recognizable in the app list.
![2021-04-28-214812_1920x1080_scrot](https://user-images.githubusercontent.com/54716352/116496822-84eca380-a86b-11eb-940a-092ec4335ba3.png)

**Comment by alienzhangyw** [comment]
OK，I understand now。But it's hard for me to design a logo，any suggestion？😂

**Comment by Botspot** [comment]
> OK，I understand now。But it's hard for me to design a logo，any suggestion？😂

Here's the icon for scratch:
![786-7861727_scratch-desktop-4-scratch-desktop-logo](https://user-images.githubusercontent.com/54716352/116582902-6c1bd680-a8db-11eb-9b4e-a451a98b1c45.png)

Here's the icon for turbowarp:
![icon-64](https://user-images.githubusercontent.com/54716352/116582654-32e36680-a8db-11eb-919d-cf066a2a9237.png)

Other similar icons I found:
![image](https://user-images.githubusercontent.com/54716352/116583149-a9806400-a8db-11eb-891f-4783d3b46159.png)
![image](https://user-images.githubusercontent.com/54716352/116583184-b2713580-a8db-11eb-8db6-bdfaf6794d99.png)

**Comment by alienzhangyw** [comment]
I released v1.2.0 with a new square logo.
![blockpi_256x256](https://user-images.githubusercontent.com/58874541/116851573-2fc2e000-ac25-11eb-9e7a-18886476c82f.png)
It has a blockly block background, a yellow python 'P', and a raspberry color 'i'.
I draw this with Windows Paint 3D 🤣.
